Discretization/extendBoundaryTensor3D.m

```diff
@@ -21,18 +21,18 @@
 xvalue(:,:,end,:) = xvalue(:,:,end-1,:);
 
 yvalue = zeros(Nx + 2, Ny + 2, Nz + 2, Tx);
-yvalue(2:end-1, 2:end-1, 2:end-1, :) = im(:,:,:,:,2);
-yvalue(1,2:end-1,2:end-1,:) = im(1,:,:,:,2);
-yvalue(end,2:end-1,2:end-1,:) = im(end,:,:,:,2);
+%yvalue(2:end-1, 2:end-1, 2:end-1, :) = im(:,:,:,:,2);
+%yvalue(1,2:end-1,2:end-1,:) = im(1,:,:,:,2);
+%yvalue(end,2:end-1,2:end-1,:) = im(end,:,:,:,2);
 yvalue(:,1,:,:) = yvalue(:,2,:,:);
```

Comment:
You are right about this. I should have used the standard format for tensors in nifti, like the output of FSL dtifit. I am going to change it!